src/utilities/sql/Test/SqlFile_GTest.cpp

```diff
@@ -73,10 +75,15 @@ TEST_F(SqlFileFixture, FuelTypes) {
   EXPECT_NEAR(409.97, *(sqlFile3.districtCoolingTotalEndUses()), 2);
   EXPECT_NEAR(378.43, *(sqlFile3.districtHeatingWaterExteriorEquipment()), 2);
   EXPECT_NEAR(378.43, *(sqlFile3.districtHeatingWaterTotalEndUses()), 2);
-  EXPECT_NEAR(346.90, *(sqlFile3.districtHeatingSteamExteriorEquipment()), 2);
-  EXPECT_NEAR(346.90, *(sqlFile3.districtHeatingSteamTotalEndUses()), 2);
-  EXPECT_NEAR(725.33, *(sqlFile3.districtHeatingExteriorEquipment()), 2);
-  EXPECT_NEAR(725.33, *(sqlFile3.districtHeatingTotalEndUses()), 2);
+  if (energyPlusBuildSHA() == "c854ba6bfe") {
+    GTEST_SKIP() << "\nFIXME: Skip the checks for DistrictHeatingSteam pending new E+ package after https://github.com/NREL/EnergyPlus/pull/10212\n";
+  } else {
+    EXPECT_TRUE(false) << "Please come remove the special check now that a new E+ package is out";
+    EXPECT_NEAR(346.90, *(sqlFile3.districtHeatingSteamExteriorEquipment()), 2);
+    EXPECT_NEAR(346.90, *(sqlFile3.districtHeatingSteamTotalEndUses()), 2);
+    EXPECT_NEAR(725.33, *(sqlFile3.districtHeatingExteriorEquipment()), 2);
+    EXPECT_NEAR(725.33, *(sqlFile3.districtHeatingTotalEndUses()), 2);
+  }
```

Comment:
Skip a portion of the test. With a check that will fail as soon as a new E+ package is out, so we don't forget to remove the special check here.

* https://github.com/NREL/EnergyPlus/pull/10212